scripts/auto-rebase/rebase.sh

```diff
@@ -770,6 +771,35 @@ update_openshift_manifests() {
     #       manifests we're manually updating them as needed for now.
     # TODO: Enable in assets.yaml and handle modifications
 
+    #-- csi-snapshot-controller ---------------------------
+    local target="${REPOROOT}/assets/components/csi-snapshot-controller/csi_controller_deployment.yaml"
+    yq -i '.metadata.namespace = "kube-system"' $target
+    yq -i '.spec.template.spec.containers[0].image = "{{ .ReleaseImage.csi_snapshot_controller }}"' $target
+    yq -i '.spec.template.spec.containers[0].args = [ "--v=2", "--leader-election=false"]' $target
+    yq -i 'del(.spec.template.spec.priorityClassName) | del(.spec.template.spec.containers[0].securityContext.seccompProfile)' $target
+    yq -i 'with(.spec.template.spec.containers[0].securityContext; .runAsUser = 65534)' $target
```

Comment:
Is that user ID special?

Reply:
No, it was arbitrary.  I just needed a user ID that was compatible with the `runAsNonRoot` security context.

Reply:
Doesn't the system allocate one automatically?